doc/ledger3.texi

```diff
@@ -4174,6 +4176,29 @@ This example causes 10% of the matching account's total to be deferred
 to the @samp{Savings} account---as a balanced virtual posting, which
 may be excluded from reports by using @option{--real}.
 
+@subsubsection Credit Card Cashback
+
+Credit cards sometimes provide a cashback percentage of purchases. This
+can be setup with the following:
+
+@smallexample @c input:C371854
+; This automated transaction will add to "Assets:Credit Card Cashback"
+; the amount of the transaction multiplied by the "cashback" tag.
+
+= "Liabilities:Credit Card" and %cashback
+    Assets:Credit Card Cashback  (-tag("cashback"))
+    Income:Credit Card Rewards    (tag("cashback"))
```

Comment:
Given that cashback is usually a certain percentage, the cashback typed metadata should be percent value, e.g. `2%` rather than a mathematical representation, e.g. `0.01`.
In order to support a percent value as suggested below the automated transaction must be changed as follows:
```suggestion
    Assets:Credit Card Cashback  (-amount * tag("cashback") * 0.01)
    Income:Credit Card Rewards    (amount * tag("cashback") * 0.01)
```